### ISSUE TITLE
link to change password

### ISSUE THREAD

**Issue by codersquid** [researchcompendia/researchcompendia#75]
There is no link for the user to change password.

**Comment by codersquid** [comment]
...that moment when you realize you didn't give users a link to change their password. facepalm.

**Comment by codersquid** [comment]
Could you take a look to make sure I didn't make a stupid change. I went ahead and deployed this already though because not giving users a way to change their password is pretty egregious.

(I don't like that doing a page refresh jumps the view back to compendia, but I'm not sure how to fix that just yet. I'll open a separate issue).

**Comment by jaseiler** [comment]
The password button works for me.  
I'm still getting sent to my account page upon login, only now with your tab system (which I do like) it looks completely blank because I have no compendia, so you might want to start on a different tab when people visit the account page. That, or put outlines on the tabs even without hovering, so it is obvious they aren't just links.

**Comment by codersquid** [comment]
I just pushed a change to make user profile the landing tab, and compendia the second. Also, the compendia tab doesn't show up if the user has no compendia.

I'll need to look at the weird tab style problem at some point. the tabs show when I run things on my local box. Maybe collectstatic didn't push something it should have pushed to s3. I opened #79 to track this.

user redirection is tracked in #67
